.github/scripts/junit.py

```diff
@@ -361,7 +361,7 @@ def split_report_path(base_path: str, report_path: str) -> Tuple[str, str]:
         failure_messages.append(f"Gradle task had a failure exit code. Failing this script.")
 
     if thread_dump_url:
-        failure_messages.append(f"Thread dump available at {thread_dump_url}. Failing this script.")
+        failure_messages.append(f"Thread dump available at {thread_dump_url} Failing this script.")
```

Comment:
it is a bit wired to have upper case "Failing" ... could you please tweak the log message?